src/Resources/app/storefront/src/checkout/checkout.plugin.js

```diff
@@ -166,6 +171,54 @@ export default class CheckoutPlugin extends Plugin {
         }
     }
 
+    renderStoredPaymentMethod(paymentMethod) {
+        //  if the container doesn't exits don't try to render the component
+        const paymentMethodContainer = $('[data-adyen-payment-method="' + this.paymentMethodTypeHandlers["oneclick"] + '"]');
+
+        // container doesn't exist, something went wrong on the template side
+        // If payment method doesn't have details, just skip it
+        if (!paymentMethodContainer) {
+            return;
+        }
+
+        //Show the payment method's contents if it's selected by default
+        if (validatePaymentMethod()) {
+            $('[data-adyen-payment-method-id]').show();
+        }
+
+        //Hide other payment method's contents when selecting an option
+        $('[name=paymentMethodId]').on("change", function () {
+            $('.adyen-payment-method-container-div').hide();
+            $(`[data-adyen-payment-method-id="${$(this).val()}"]`).show();
+        });
+
+        /*Use the storedPaymentMethod object and the custom onChange function as the configuration object together*/
+        const configuration = Object.assign(paymentMethod, {
+            onChange: this.onPaymentMethodChange.bind(this)
+        });
+
+        if (paymentMethod.type === 'scheme') {
+            configuration.enableStoreDetails = true;
+        }
```

Comment:
Is it needed for oneclick payment methods?
I think it's only required for payment methods where you do the oneoff payment and using that payment you create the token at Adyen. Therefore for already stored payment methods I believe this config is not needed.

```suggestion
```